### DISCUSSION TITLE
Dotnet Conference 2024 Demo code

### DISCUSSION THREAD

**Discussion by coderdnewbie** [microsoft/fluentui-blazor#2937] [General]
To help the community, I have improved the demo code from the Dotnet Conference 2024 to fix an issue I saw with the theming which caused an error in the console log, and I also upgraded this to dotnet 9.0, so it uses Assets and MapStaticAssets().

The code is in this github repository:

https://github.com/coderdnewbie/DemoFluentUI

Thanks for doing a great presentation at Dotnet Conf 2024.

Jay Tandon

**Comment by vnbaaij** [comment]
Thanks for the compliment and thanks for the fixes. Could you please create a PR in the Conf samples repo so we can update the original?

**Comment by coderdnewbie** [comment]
I have made the PR as requested in the Conf Samples repo. If there are any issues, please let me know, happy to implement changes. Note: I have only made code changes, and not the documentation.

**Comment by coderdnewbie** [comment]
The demo is upgraded to .Net 9.0. Closing this issue
